src/common/config_opts.h

```diff
@@ -514,11 +514,10 @@ OPTION(mds_bal_split_rd, OPT_FLOAT, 25000)
 OPTION(mds_bal_split_wr, OPT_FLOAT, 10000)
 OPTION(mds_bal_split_bits, OPT_INT, 3)
 OPTION(mds_bal_merge_size, OPT_INT, 50)
-OPTION(mds_bal_merge_rd, OPT_FLOAT, 1000)
-OPTION(mds_bal_merge_wr, OPT_FLOAT, 1000)
 OPTION(mds_bal_interval, OPT_INT, 10)           // seconds
 OPTION(mds_bal_fragment_interval, OPT_INT, 5)      // seconds
 OPTION(mds_bal_fragment_size_max, OPT_INT, 10000*10) // order of magnitude higher than split size
+OPTION(mds_bal_fragment_fast_limit, OPT_FLOAT, 1.5) // multiple of size_max that triggers immediate split
```

Comment:
How about mds_bal_fragment_fast_factor instead, since it's a multiplier and not an absolute value?